container/config.ini

```diff
@@ -18,7 +18,7 @@ cpe-oval = /usr/share/openscap/cpe/openscap-cpe-oval.xml
 ssg = /usr/share/xml/scap/ssg/content
 
 [CVEScanner]
-fetch-cve = yes
+fetch-cve = no
```

Comment:
With `fetch-cve = no` wouldn't it be necessary to build the container every time new CVE content is released?
Why make it necessary to build it every time?